okhttp/src/main/java/okhttp3/WebSocketStatusCode.java

```diff
@@ -0,0 +1,114 @@
+package okhttp3;
+
+/**
+ * Status code as defined by
+ * <a href="http://tools.ietf.org/html/rfc6455#section-7.4">
+ * Section 7.4 of RFC 6455
+ * </a>.
+ */
+public class WebSocketStatusCode {
```

Comment:
`final`?

Reply:
and private constructor, and two-space indent